go.mod

```diff
@@ -1,87 +1,81 @@
 module github.com/spacelift-io/spacelift-operator
 
-go 1.21
-
-toolchain go1.21.4
+go 1.23.2
```

Comment:
I wanted this to be `go 1.23` but the emoji lib for some reason requires 1.23.2, so `go mod tidy` autobumps this to that version.
https://github.com/tmdvs/Go-Emoji-Utils/blob/main/go.mod#L3